packages/governance/src/types.js

```diff
@@ -153,7 +223,35 @@
  */
```

Comment:
I was surprised to see a parameter named `seat` of type `Handle<'Voter'>` rather than `UserSeat` or `ZCFSeat`. The latter two are far objects with methods, whereas this one is just a handle.

Reply:
![image](https://user-images.githubusercontent.com/273868/131570194-f0b46887-5831-4c44-b244-48c000f322d1.png)
That's interesting. When I try typing `submitVote` as `SubmitVote`, TS complains of a type error. It seems the code is assuming that the completedBallot is a `CompleteEqualWeightBallot` whereas TS doesn't know that. I have no idea what should be done about that.